tensorboard/main.py

```diff
@@ -261,6 +261,13 @@ def main(unused_argv=None):
     efi.inspect(FLAGS.logdir, event_file, FLAGS.tag)
     return 0
   else:
+
+    # The default is HTTP/1.0 for some strange reason. If we don't use
```

Comment:
FWIW, the BasicHTTPRequestHandler docs say it's to keep people from shooting themselves in the foot by omitting Content-Length:
https://docs.python.org/2/library/basehttpserver.html#BaseHTTPServer.BaseHTTPRequestHandler.protocol_version

But yeah, it seems like HTTP/1.1 should be the default and BaseHTTPRequestHandler should just die on you if you attempt to send a response with no content length.

Reply:
I agree.